app/components/atoms/number-heading/number-heading.styles.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+import polygon from './img/Polygon.svg';
+
+export interface NumberHeadingProps {
+  children?: string;
```

Comment:
`children?: ReactNode`, ReactNode returns the value of the component :D see: https://github.com/typescript-cheatsheets/react

👍🏽 

You can just use `children: "a"` in the args of the story.